packages/rocketchat-theme/assets/stylesheets/base.less

```diff
@@ -4031,10 +4031,6 @@ body:not(.is-cordova) {
 			}
 		}
 
-		.select-arrow {
```

Comment:
As we discussed before I will add `select-arrow` to all `select` and make it global in another PR.